### PR TITLE
Update getSystem to try both http and https

### PR DESCRIPTION
**Pull request by Floyer007**
The newest TV software (API 6 at least) just allows https connections to get system information.

### CODE REVIEW

File: haphilipsjs/__init__.py [hunk 1]
```diff
@@ -747,13 +747,21 @@ def _decodeSystem(self, system) -> SystemType:
...
-            self.name = r.get("name")
-        else:
-            self.system = {}
-            self.name = None
+        # Newest TV software requires https for system-info. Therefore we will try both protocols.
+        protocols = ["http", "https"]
+        orig_protocol = self.protocol
+        
+        for prot in protocols:
+            self.protocol = prot
```
Comment (elupus) [inline haphilipsjs/__init__.py]: Please add and optional parameter to getReq for protocol (and forward to _url() function, then we don't need this back and forth.

Comment (elupus) [reply haphilipsjs/__init__.py]: Also it should start with currently set protocol and fallback.

Comment (Floyer007) [reply haphilipsjs/__init__.py]: Added the getReq paramter in [717445c](https://github.com/danielperna84/ha-philipsjs/pull/33/commits/717445c8367dbbbee1b4b1fe85895fc71faa8307)
The other will follow the next days.

Comment (Floyer007) [reply haphilipsjs/__init__.py]: Starts with currently set protocol now and falls back afterward.
Also the protocol is not set globally by the getSystem function anymore.

**Review by elupus — APPROVED** [review]

**Pull request merged**